apps/antalmanac/src/components/RightPane/SectionTable/GradesPopup.tsx

```diff
@@ -31,12 +30,16 @@ async function getGradeData(
      *
      * @example { gradeACount: 10, gradeBCount: 20 }
      */
+    const totalGrades = Object.values(Object.entries(courseGrades).filter(([key]) => key !== 'averageGPA')).reduce(
+        (acc, [_, value]) => acc + value,
+        0
+    );
```

Comment:
I believe the JSDoc comment refers to `grades`, so we should shift around the comment (or the code), whichever you prefer.